resources/puppetlabs/lein-ezbake/template/global/ext/cli/reload.erb

```diff
@@ -14,7 +14,7 @@ if [ -r "$restartfile" ];  then
 
         ((timeout--))
         if [ "$timeout" = 0 ]; then
-            echo "Reload timed out after <%= EZbake::Config[:start_timeout] %> seconds"
+            echo "$timeout" | awk '{printf "Reload timed out after %.3f secs\n", $1/10}'
```

Comment:
omg 

also i think that i did not see that comment when I made that change, github must not update very quickly for me